File: Domain/Blog.cs

namespace Domain
{
    public class Blog
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public DateTime Date { get; set; }
        public string Description { get; set; }
        public string Category { get; set; }
        public string Body { get; set; }
        public string UrlSuffix { get; set; }
        public ICollection<Photo> Photos { get; set; }
        public AppUser AppUser { get; set; }
        public ICollection<Comment> Comments { get; set; } = new List<Comment>();
    }
}

File: client/src/app/models/blog.ts
import { Photo } from "./photo";
import { User } from "./user";

export interface Blog {
    id: string;
    title: string;
    date: Date;
    description: string;
    category: string;
    body: string;
    urlSuffix: string;
    appUser?: User;
    photos?: Photo[];
}
